Resources/services.xml

```diff
@@ -28,5 +30,28 @@
                  class="SwagTax\Subscriber\BasicSettingsTaxSubscriber">
             <tag name="shopware.event_subscriber"/>
         </service>
+
+        <service id="SwagTax\Components\ThirdPlugins\ThirdPluginUpdater"
+                 class="SwagTax\Components\ThirdPlugins\ThirdPluginUpdater">
+            <argument type="service" id="SwagTax\Components\ThirdPlugins\ThirdPluginHandlerFactory"/>
+        </service>
+
+        <service id="SwagTax\Components\ThirdPlugins\ThirdPluginHandlerFactory"
+                 class="SwagTax\Components\ThirdPlugins\ThirdPluginHandlerFactory">
+            <argument type="tagged" tag="swag_tax.plugin_handler"/>
```

Comment:
you have to check this in the min Shopware version of the plugin. because we are using different Symfony versions and I don't know if this works for all the same

Reply:
Ohh yes sure.. i change it to a event